docs/troubleshooting/authentication/code_403_the_caller_does_not_have_permission_status_permission_denied.md

```diff
@@ -0,0 +1,40 @@
+---
+keywords: ['permission', 'permission_denied', 'code']
+slug: /code-403-the-caller-does-not-have-permission-status-permission-denied
+title: 'Code: 403, The caller does not have permission. status: PERMISSION_DENIED'
+---
+# Code: 403, The caller does not have permission. status: PERMISSION_DENIED
+
+This error typically occurs when your application or service account does not have the required permissions to access a resource in Google Cloud or Firebase.
+
+## What does this error mean?
+
+You may encounter this error due to one or more of the following reasons:
```

Comment:
Let's have the error message just before this line and remove the Example Full Error Message section because its gonna be redundant. 

The h2 title can also be called "Code 403 Error Message" instead of "What does this error mean"